go/v1/api/validators/validationlib/basic_check_test.go

```diff
@@ -0,0 +1,156 @@
+// Copyright 2018 The Grafeas Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package validation
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestIsAlpha(t *testing.T) {
+	tests := []struct {
+		input   string
+		isalpha bool
+	}{
+		{
+			input:   strings.Repeat("a", 0),
+			isalpha: true,
+		},
+		{
+			input:   strings.Repeat("a", 100),
+			isalpha: true,
+		},
+		{
+			input:   strings.Repeat("a", 1000000),
+			isalpha: true,
+		},
+		{
+			input:   "你好",
+			isalpha: false,
+		},
+		{
+			input:   "\b5Ὂg̀9! ℃ᾭG",
+			isalpha: false,
+		},
+	}
+
+	for _, tt := range tests {
+		if IsAlpha(tt.input) != tt.isalpha {
```

Comment:
This can be simplified into something like:

```
if isAlpha := IsAlpha(tt.input); isAlpha != tt.isalpha {
  t.Errorf("IsAlpha(%q) = %t, want %t", tt.input, isAlpha, tt.isAlpha)
}
```

Reply:
done.